app/assets/javascripts/views/map_home_view.js

```diff
@@ -8,13 +8,34 @@
 
     defaults: {
       buckets: {
```

Comment:
There is a way to do this that I think is more clear that is to have one object with the colors for each target inside the object buckets. 
Something like: 

```
buckets: {
  anemia: {
    //Colors for anemia
  },
  stunting: {
    //Colors for stungting 
 }...
}